spec/requests/api/instances_spec.rb

```diff
@@ -21,52 +25,105 @@ def app
     end
   end
 
-  context "Instance terminate action" do
-    let(:instance) { FactoryGirl.create(:vm_openstack) }
-    let(:instance1) { FactoryGirl.create(:vm_openstack) }
-    let(:instance2) { FactoryGirl.create(:vm_openstack) }
+  context "Instance actions" do
```

Comment:
@abellotti each action still has its own group - I just introduced this higher-level context to separate their shared setup from the existing tests in this file that don't need them